NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs

```diff
@@ -125,34 +120,44 @@ private AccountSettingsDialog(Gtk.Builder builder, AccountSettingsDialogControll
                 }
             }
         };
-        _customSymbolText.OnNotify += (sender, e) =>
+        _customSymbolRow.OnNotify += (sender, e) =>
         {
             if (e.Pspec.GetName() == "text")
             {
+                if(_customSymbolRow.GetText().Length > 3)
+                {
+                    _customSymbolRow.SetText(_customSymbolRow.GetText().Substring(0, 3));
+                    _customSymbolRow.SetPosition(-1);
+                }
                 if (!_constructing)
                 {
                     Validate();
                 }
             }
         };
-        _customCodeText.OnNotify += (sender, e) =>
+        _customCodeRow.OnNotify += (sender, e) =>
         {
             if (e.Pspec.GetName() == "text")
             {
+                if(_customCodeRow.GetText().Length > 3)
+                {
+                    _customCodeRow.SetText(_customCodeRow.GetText().Substring(0, 3));
+                    _customCodeRow.SetPosition(-1);
+                }
                 if (!_constructing)
                 {
                     Validate();
                 }
             }
         };
-        _customDecimalSeparatorDropDown.SetModel(Gtk.StringList.New(new string[3] { ".", ",", _p("DecimalSeparator", "Other") }));
-        _customDecimalSeparatorDropDown.OnNotify += (sender, e) =>
+        _customDecimalSeparatorRow.SetModel(Gtk.StringList.New(new string[3] { ". ", ", ", _p("DecimalSeparator", "Other") }));
```

Comment:
Any reason this isn't defined in the Blueprint?

Reply:
No, no reason. I'll move it to blueprint.